.ci/project-dependencies.yaml

```diff
@@ -52,6 +52,7 @@ dependencies:
             target: 7.x
       exclude:
         - kiegroup/droolsjbpm-knowledge          
+        - kiegroup/kie-jpmml-integration
```

Comment:
This was the only change required as there is already `mapping:` as part of `project: kiegroup/kie-jpmml-integration`

Reply:
yeah, i think at some point https://github.com/kiegroup/kie-jpmml-integration/commit/72ab8bfcc6b31f979f4bbb4e4cd440a4494fefe5 was moving this repository to Drools 8.x and then reverted, but branches stayed so i think that is overlooked issue

Reply:
@mareknovotny I'm not clear about you said `and then reverted`, but kie-jpmml-integration 7.x should be used for drools 7.x FDB, right?